pages/vi/find-pi.md

```diff
@@ -34,6 +34,7 @@ You can stop this process with `Ctrl`+`C`.
 As you can see here its local IP address is `192.168.0.19`.
 
 ## Method 3: nmap command
+Note: If this is your first time using the CLI on macOS you need to install brew, a package management system for the CLI
```

Comment:
Make sure that the note is bolded with surrounding asterisks  and  "Note" is capitalized. Look at line 5 for an example.